src/main/java/mate/academy/dao/impl/MovieDaoImpl.java

```diff
@@ -45,6 +46,18 @@ public Optional<Movie> get(Long id) {
 
     @Override
     public List<Movie> getAll() {
-        return null;
+        Session session = null;
+        try {
+            session = HibernateUtil.getSessionFactory().openSession();
+            Query<Movie> getAllCommentsQuerry
+                    = session.createQuery("FROM Movie", Movie.class);
+            return getAllCommentsQuerry.getResultList();
+        } catch (Exception e) {
+            throw new RuntimeException("Can't get all movies");
```

Comment:
Do not ignore the caught exception. Include it as a cause of the DataProcessingException.